EvidenceApi/DocumentTypes.json

```diff
@@ -211,5 +211,31 @@
         "description": "All sections and additional information must be provided and signed by all applicants, photo of all pages of the booklet"
       }
     ]
+  },
+  {
+    "name": "Admission and Pupil Benefits",
+    "id": "9",
+    "documentTypes": [
+      {
+        "id": "proof-of-address",
+        "title": "Proof of Address",
+        "description": "Two valid documents to provide proof of address. It must inculde the name of the Parent who is applying clearly wittren on the document provided."
+      },
+      {
+        "id": "proof-of-child-dob",
+        "title": "Proof of Child's Date of Birth",
+        "description": "A valid document to provide proof of child's identity. Picture of child's full name and date of birth must be in the Evidence."
+      },
+      {
+        "id": "proof-of-arrival",
+        "title": "Proof of Child's Date of Birth",
```

Comment:
Should this be `Proof of Child's Date of Arrival`?

Reply:
I've updated this to `Proof of Arrival`, as it is in the spreadsheet.